### DISCUSSION TITLE
Adding pointer to every component

### DISCUSSION THREAD

**Discussion by him-raw99** [mantinedev/mantine#7198] [Feature requests]
hi guys 
let me begin by saying amazing work with mantine, its an amazing library with easy learning curve with all the features i could ask for

in this particular feature i would like to request to have a `pointer` prop to every component like we have in input components. because in practice this is something that i have to use inline css for the most and i believe this can be fixed quite neatly if we had pointer prop for every component. 

i am fairly new dev and already felt overwhelmed looking at the code. i hope to make this contribution on my own if you'll allow me, since i think its pretty small change (i could be wrong💀💀) 

thanks

**Comment by rtivital** [comment]
Cursor: pointer is set automatically on interactive components, if you want to extend it to inputs that do not have browser pointer cursor by default, see https://mantine.dev/theming/theme-object/#cursortype.

If you want to add cursor pointer to other mantine components, use Styles API in theme object – https://mantine.dev/styles/styles-api/#classnames-in-themecomponents
